dts/bindings/input/gpio-keys.yaml

```diff
@@ -35,15 +35,23 @@ properties:
        Debouncing interval time in milliseconds.
        If not specified defaults to 30.
 
+  no-interrupts:
```

Comment:
Nit: Rename to reflect polling behavior, maybe `poll_pins`?  The description can mention interrupts are disabled. Or maybe reverse the logic to name this `use_interrupts`?  I'm not a fan of putting `no` prefix on booleans :-)

Reply:
Yeah, it's a bit silly to define something for what it's not, my reasoning was that a user would try the default, get a "interrupt configuration failed" message and then it's somewhat obvious that "no interrupt" is what you want. Don't know, maybe `polling-mode`? @kartben more ideas? I'm not too attached to any name here but I can't invert the logic because of how boolean works in devicetree unfortunately.

Reply:
+1 to `polling-mode`

Reply:
oky, let me change that